### ISSUE TITLE
[Bug]: Required dependency lack of ProtoBuf

### ISSUE THREAD

**Issue by XuanYang-cn** [milvus-io/pymilvus#1077]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

```
Traceback (most recent call last):
  File "/home/yangxuan/Github/pymilvus/examples/hello_milvus.py", line 12, in <module>
    from pymilvus import (
  File "/home/yangxuan/.virtualenvs/test-protobuf/lib/python3.9/site-packages/pymilvus/__init__.py", line 13, in <module>
    from .client.stub import Milvus
  File "/home/yangxuan/.virtualenvs/test-protobuf/lib/python3.9/site-packages/pymilvus/client/stub.py", line 3, in <module>
    from .grpc_handler import GrpcHandler
  File "/home/yangxuan/.virtualenvs/test-protobuf/lib/python3.9/site-packages/pymilvus/client/grpc_handler.py", line 12, in <module>
    from ..grpc_gen import milvus_pb2_grpc
  File "/home/yangxuan/.virtualenvs/test-protobuf/lib/python3.9/site-packages/pymilvus/grpc_gen/milvus_pb2_grpc.py", line 5, in <module>
    from . import common_pb2 as common__pb2
  File "/home/yangxuan/.virtualenvs/test-protobuf/lib/python3.9/site-packages/pymilvus/grpc_gen/common_pb2.py", line 5, in <module>
    from google.protobuf.internal import enum_type_wrapper
ModuleNotFoundError: No module named 'google.protobuf'
```

### Environment details

```markdown
- Hardware/Softward conditions (OS, CPU, GPU, Memory):
- Method of installation (Docker, or from source):
- Milvus version (v0.3.1, or v0.4.0):
- Milvus configuration (Settings you made in `server_config.yaml`):
```

**Comment by nameczz** [comment]
Just in case anyone else encounters this problem.

pip3 install protobuf==3.20.0  can temporarily solve this problem.

**Comment by XuanYang-cn** [comment]
fixed at pymilvus-2.1.1.dev2

will be available in pymilvus 2.1.1
